pkg/testutil/rand/model.go

```diff
@@ -129,76 +129,80 @@ func WithMessageFormat() ModelMessageOption {
 	}
 }
 
-// ------------------Messages------------------
+// ------------------Translation-----------------
 
-// modelMessages generates a random model.Messages with the given
-// count of Messages.messages and using the provided options for each message.
-func modelMessages(msgCount uint, msgOpts ...ModelMessageOption) *model.Messages {
-	messages := &model.Messages{
+// modelTranslation generates a random model.Translation with the given
+// count of messages and using the provided options for each message.
+func modelTranslation(msgCount uint, msgOpts ...ModelMessageOption) *model.Translation {
+	translation := &model.Translation{
 		Language: Language(),
 		Original: gofakeit.Bool(),
 		Messages: make([]model.Message, msgCount),
 	}
 
 	if msgCount == 0 {
-		return messages
+		return translation
 	}
 
 	msgs := ModelMessageSlice(msgCount, msgOpts...)
```

Comment:
Can `ModelMessageSlice` be renamed to `ModelMessages`?

Reply:
what about modelServiceSlice ?

Reply:
Ideally, it is `modelServices`.